### ISSUE TITLE
Zoxide issue with login xonsh and a workaround

### ISSUE THREAD

**Issue by eugenesvk** [dyuri/xontrib-zoxide#1]
FYI: there is an issue with zoxide in [xonsh login shells](https://github.com/xonsh/xonsh/issues/4151) that requires using this:
```
with ${...}.swap(UPDATE_OS_ENVIRON=True):
  execx($(zoxide init xonsh), 'exec', __xonsh__.ctx, filename='zoxide')
```
instead of the regular one-liner
Not sure if you'd want to update your xontrib or wait for the [upstream fix](https://github.com/ajeetdsouza/zoxide/issues/157)

**Comment by dyuri** [comment]
No problem at all, 0.2.0 "released" with the fix. :)

**Comment by ajeetdsouza** [comment]
This issue has been fixed in zoxide 0.6.0. Thanks!
